src/main/java/com/znsio/teswiz/runner/Setup.java

```diff
@@ -133,7 +133,7 @@ static String getCurlProxyCommand() {
         return curlProxyCommand;
     }
 
-    static List<String> getExecutionArguments() {
+    static List<String> getExecutionArguments() throws IOException {
```

Comment:
why are we throwing exceptions? Throw a custom runtime exception

Reply:
Removed it got added because of method chaining, have fixed the method from where it was getting generated.